cloudvolume/cloudvolume.py

```diff
@@ -160,6 +157,27 @@ def __init__(self, cloudpath, mip=0, bounded=True, autocrop=False, fill_missing=
 
     self.pid = os.getpid()
 
+  @classmethod
+  def from_numpy(cls, arr, vol_path='file://tmp/'+generate_random_string(),
+                  resolution=(4,4,40), voxel_offset=(0,0,0), 
+                  chunk_size=(64,64,64)):
```

Comment:
Maybe the chunk size should be (128,128,64) since (64,64,64) has typically been terrible for transfer rates.